client/src/pages/login.js

```diff
@@ -1,12 +1,99 @@
 import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Api from '../services/api';
+import '../styles/login.css';
 
 
 class Login extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+      errorMessage: '',
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  // eslint-disable-next-line react/sort-comp
+  validateForm() {
+    const { email } = this.state;
+    const { password } = this.state;
+    return email.length > 0 && password.length > 7;
+  }
+
+  handleChange(event) {
+    this.setState({
+      [event.target.id]: event.target.value,
+    });
+  }
+
+  getLogedin() {
+    Api.getLogedin(this.state.email, this.state.password)
+      .then((response) => {
+        if (response.status >= 200 && response.status < 300) {
+          localStorage.setItem('Bearer ', response.data);
+          // eslint-disable-next-line react/prop-types
+          this.props.history.push('/');
+        } else if (response.error.status === 403) {
+          this.setState({ errorMessage: 'Email or password incorrect' });
+        }
+      }).catch((error) => {
+        this.setState({ errorMessage: error.message });
+      });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    // eslint-disable-next-line no-undef
+    this.getLogedin();
+  }
```

Comment:
line 34: use getLog**g**ed**I**n;
line 37: I am not sure if we can use http status codes above '200' here
line 42: 'Email or password **are** incorrect'
line 45: `errorMessage` should be clear for user
line 50: could you explain what `event.preventDefault()` here is for, please?
line 58: we use `<Container>` in our app in similar cases.

Reply:
25 ; ok fixed
37: we can why not ?
45:changed to Invalid password or email
50: https://developer.mozilla.org/ru/docs/Web/API/Event/preventDefault

Reply:
line 37: We get only `200` http status when authentication is successful, or not?
line 50: I want to know what is going on here :), and I have seen web information about this already

Reply:
By default html form send GET request after it submitted. `preventDefault` disables this unnecessary functionality